src/instructlab/eval/mmlu.py

```diff
@@ -122,8 +122,14 @@ def __init__(
         self.batch_size = batch_size
         self.device = device
 
-    def _run_mmlu(self) -> dict:
-        model_args = f"pretrained={self.model_path},dtype={self.model_dtype}"
+    def _run_mmlu(self, server_url: str | None = None) -> dict:
+        if server_url is not None:
```

Comment:
should we do some kind of basic validation on the provided `server_url` before using it?

Reply:
Four answers:
- No because it's just a passthrough.  Can let the actual consumer (lm_eval) do the validation.
- Yes because we are adding /completions to it. This is required as of lm_eval 0.4.4.
- No because we aren't validating server_url anywhere else.
- Yes, but maybe we should do it in a follow-up and do it consistently across all of eval and maybe instructlab

Reply:
I think I agree with point 4, that kind of automatically addresses all other points